cpu/esp32/Makefile.include

```diff
@@ -93,6 +89,7 @@ CFLAGS  += -DLOG_TAG_IN_BRACKETS
 CFLAGS  += -Wno-unused-parameter -Wformat=0
 CFLAGS  += -mlongcalls -mtext-section-literals -fstrict-volatile-bitfields
 CFLAGS  += -fdata-sections -fzero-initialized-in-bss
+CFLAGS  += -ffunction-sections
```

Comment:
I would prefer to place it in front of `-fdata-sections` in the line above.

Reply:
Yep I will do.